[2025-05-05T10:08:02.689+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Pipeline-proyecto2.training_with_gradient_boosting_model manual__2025-05-05T10:06:17.235188+00:00 [queued]>
[2025-05-05T10:08:02.710+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Pipeline-proyecto2.training_with_gradient_boosting_model manual__2025-05-05T10:06:17.235188+00:00 [queued]>
[2025-05-05T10:08:02.710+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-05-05T10:08:02.753+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): training_with_gradient_boosting_model> on 2025-05-05 10:06:17.235188+00:00
[2025-05-05T10:08:02.773+0000] {standard_task_runner.py:57} INFO - Started process 126 to run task
[2025-05-05T10:08:02.781+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Pipeline-proyecto2', 'training_with_gradient_boosting_model', 'manual__2025-05-05T10:06:17.235188+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/dag-proyecto2.py', '--cfg-path', '/tmp/tmp67_pk2zf']
[2025-05-05T10:08:02.787+0000] {standard_task_runner.py:85} INFO - Job 52: Subtask training_with_gradient_boosting_model
[2025-05-05T10:08:02.965+0000] {task_command.py:410} INFO - Running <TaskInstance: Pipeline-proyecto2.training_with_gradient_boosting_model manual__2025-05-05T10:06:17.235188+00:00 [running]> on host 3ffd6f9b5180
[2025-05-05T10:08:03.282+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Pipeline-proyecto2' AIRFLOW_CTX_TASK_ID='training_with_gradient_boosting_model' AIRFLOW_CTX_EXECUTION_DATE='2025-05-05T10:06:17.235188+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-05T10:06:17.235188+00:00'
[2025-05-05T10:08:08.489+0000] {logging_mixin.py:149} WARNING - 2025/05/05 10:08:08 INFO mlflow.tracking.fluent: Experiment with name 'Gradient_Boosting_looking_for_the_best_model' does not exist. Creating a new experiment.
[2025-05-05T10:08:16.229+0000] {logging_mixin.py:149} WARNING - 2025/05/05 10:08:16 INFO mlflow.tracking.fluent: Autologging successfully enabled for sklearn.
[2025-05-05T10:08:17.327+0000] {logging_mixin.py:149} WARNING - 2025/05/05 10:08:17 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2025-05-05T10:08:24.057+0000] {logging_and_warnings.py:72} WARNING - /home/***/.local/lib/python3.9/site-packages/joblib/parallel.py:1378: UserWarning: Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1
  n_jobs = self._backend.configure(

[2025-05-05T10:26:26.551+0000] {credentials.py:1123} INFO - Found credentials in environment variables.
[2025-05-05T10:26:28.272+0000] {logging_mixin.py:149} WARNING - 2025/05/05 10:26:28 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/***/.local/lib/python3.9/site-packages/mlflow/models/signature.py:137: UserWarning: Hint: Inferred schema contains integer column(s). Integer columns in Python cannot represent missing values. If your input data contains missing values at inference time, it will be encoded as floats and will cause a schema enforcement error. The best way to avoid this problem is to infer the model schema based on a realistic data sample (training dataset) that includes missing values. Alternatively, you can declare integer columns as doubles (float64) whenever these columns may have missing values. See `Handling Integers With Missing Values <https://www.mlflow.org/docs/latest/models.html#handling-integers-with-missing-values>`_ for more details."
[2025-05-05T10:26:33.252+0000] {logging_mixin.py:149} WARNING - 2025/05/05 10:26:33 WARNING mlflow.utils.autologging_utils: MLflow autologging encountered a warning: "/home/***/.local/lib/python3.9/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils."
[2025-05-05T10:26:39.174+0000] {logging_mixin.py:149} WARNING - 2025/05/05 10:26:39 INFO mlflow.sklearn.utils: Logging the 5 best runs, 11 runs will be omitted.
[2025-05-05T10:26:40.701+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-05T10:26:40.730+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=Pipeline-proyecto2, task_id=training_with_gradient_boosting_model, execution_date=20250505T100617, start_date=20250505T100802, end_date=20250505T102640
[2025-05-05T10:26:40.817+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-05T10:26:40.865+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
